README.md

```diff
@@ -5,17 +5,34 @@ You can view the latest rendered build of this content at:
 
     http://docs.couchdb.org/en/latest
 
-# Building this repo
+# Getting Started
 
-Install Python 2.7+ and pip. Then:
+## Prerequisites
+
+- Python 3.x
+- Pip
+
+### Preparing Prerequisites using [asdf](https://asdf-vm.com/#/)
```

Comment:
I'd rather not give this "top billing."

Would you be willing to restore the virtualenv-based way of setting this up, and add this as the secondary way below it?

Reply:
Of course @wohali  - i thought `virtualenv` wouldn't be used anymore since the `venv`-directory is missing. I will revert that.

Reply:
@wohali nevermind - i made a mistake.

Reply:
Here's what I'd recommend for updating the instructions to:

```
python3 -m venv .venv
source .venv/bin/activate
pip install -r requirements.txt
```

Reply:
@wohali - done. Sorry for the noise.